Rakefile

```diff
@@ -329,3 +329,40 @@ namespace :docs do
     sh "mv #{docs_name}-#{version}.gem pkg"
   end
 end
+
+task :analysis do
+  require "jekyll/utils/ansi"
+  require "open3"
+
+  cmd = [
+    "docker", "run", "--rm", "--env=CODE_PATH=#{Dir.pwd}", \
+    "--volume='#{Dir.pwd}:/code'", "--volume=/var/run/docker.sock:/var/run/docker.sock", \
+    "--volume=/tmp/cc:/tmp/cc", "-i", "codeclimate/codeclimate", "analyze"
```

Comment:
AFAIW there isn't because all their stuff requires Docker, actually this is the officially sanctioned way to run CodeClimate locally by CodeClimate.  Except I shivved it so that we could generate a file with the analysis to view in our editors.